frontend/packages/console-app/src/components/cloud-shell/CloudShellTerminal.tsx

```diff
@@ -27,12 +28,25 @@ type Props = {
 
 type CloudShellTerminalProps = StateProps & Props;
 
+const isAdmin = (user: UserKind): boolean => {
+  if (!user.groups) {
+    return false;
+  }
+  for (const group of user.groups) {
+    if (group === 'system:cluster-admins') {
```

Comment:
I don't believe this is a reliable check for system admin. RBAC is more nuanced than this. You can have admin privileges without being in this group.

Is there a specific permission we are worried about? It would be better to do a self-subject access review.

Reply:
Previously, we tested `pods:write` in `openshift-operators` namespace with self-subject access review. @spadgett Do you have any concerns about returning back to that check? Any suggestion which permission is better to check?

Reply:
I think that would be a better check. We might want to use a different namespace like `openshift-terminal` or maybe `default`.

Reply:
I just switched them to use `openshift-terminal`. I thought there might be issues if the openshift-terminal namespace doesn't exist but everything seems to be working fine

Reply:
> I just switched them to do a subject access review on the `openshift-terminal` namespace. I thought there might be issues if the openshift-terminal namespace doesn't exist but everything seems to be working fine

If you have permission to create pods in any namespace, then the RBAC check will pass even if the namespace doesn't exist yet.